moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -135,6 +135,23 @@ Servo::Servo(const rclcpp::Node::SharedPtr& node, std::shared_ptr<const servo::P
     joint_name_to_index_maps_.insert(
         std::make_pair<std::string, JointNameToMoveGroupIndexMap>(std::string(sub_group_name), std::move(new_map)));
   }
+
+  const moveit::core::JointModelGroup* joint_model_group =
+      robot_state->getJointModelGroup(servo_params_.move_group_name);
+  const std::vector<std::string> joint_names = joint_model_group->getActiveJointModelNames();
+  const moveit::core::JointBoundsVector joint_bounds = joint_model_group->getActiveJointModelsBounds();
+  joint_acceleration_limit = -1.0;
+  for (const auto& bound : joint_bounds)
+  {
+    for (const auto& b1 : *bound)
+    {
+      double max_acceleration = servo_params_.scale.joint_acceleration *
+                                std::max(std::abs(b1.min_acceleration_), std::abs(b1.min_acceleration_));
```

Comment:
even if this were the correct math, I think this is a typo:

```suggestion
                                std::max(std::abs(b1.min_acceleration_), std::abs(b1.max_acceleration_));
```

... but again, this would of course depend on whether the commanded acceleration is in the positive or negative direction, so I think the logic should be flipped:

* First check whether the joint needs to move in the positive or negative direction
* THEN pick either min or max acceleration

Reply:
This is now handled in the new `jointLimitAccelerationScalingFactor` method.